src/app/containers/Include/__snapshots__/index.test.jsx.snap

```diff
@@ -19,11 +19,13 @@ exports[`IncludeContainer should render a fallback for VJs on an Amp page when i
       class="StyledEmbedError-sc-1oc1egd-0 fyPX"
     >
       <div
-        class="StyledErrorMessage-sc-1oc1egd-1 eCvEfh"
+        class="StyledErrorMessage-sc-1oc1egd-1 jAchtl"
       >
-        <strong>
-          Sorry, we can’t display this part of the story on this lightweight mobile page.
-        </strong>
+        <div>
```

Comment:
Updating the embed-error component to the latest version has changed this snapshot, https://github.com/bbc/psammead/blob/156815c128f560819c1bacee82bda71e4522a06c/packages/components/psammead-embed-error/src/index.jsx#L71